ksql-parser/src/test/java/io/confluent/ksql/schema/ksql/LogicalSchemasTest.java

```diff
@@ -97,8 +98,12 @@ public void shouldHaveTestsForAllTypes() {
 
   @Test
   public void shouldGetLogicalForEverySqlType() {
-    SQL_TO_LOGICAL.forEach((sqlType, logical) ->
-        assertThat(LogicalSchemas.fromSqlTypeConverter().fromSqlType(sqlType), is(logical)));
+    for (Entry<Type, Schema> entry : SQL_TO_LOGICAL.entrySet()) {
```

Comment:
Personally found the for-each loop easier to read. To each their own?

Reply:
It has more to do with debug-ability than with readability. Stepping through, inspecting internals and modifying state is really annoying with java8 forEach. Maybe I'm missing something fundamental in the debugger tools... for this specific case though, I guess just setting a condition breakpoint would've worked

Reply:
What IDE are you using? IntelliJ will let you set a breakpoint in the lambda....

Reply:
Setting a breakpoint in the lambda is no problem. It's stepping over/in/around after I've paused execution that is wonky.